t/05-scheduler-dependencies.t

```diff
@@ -35,6 +35,14 @@ use Test::Mojo;
 use Test::More;
 use Test::Warnings;
 
+# This enforces order in the processing of the nodes, to test PR#1623 on test 191 and subtest 19 from test 197
+no warnings 'redefine';
+*OpenQA::Schema::Result::Jobs::search_for = sub {
```

Comment:
isn't this a job for Test::MockModule?

Reply:
Why? it's acting in the same way in this case

Reply:
It's better to store the original function and wrap it to avoid loosing the original code

Reply:
@coolo @mudler please check the latest version :-)